src/main/java/net/mcreator/ui/component/UnsupportedComponent.java

```diff
@@ -31,6 +31,7 @@
 
 public class UnsupportedComponent extends JPanel {
 
+	// helper method that marks provided component as not supported

```

Comment:
Comments above methods should be JavaDOC so they are picked by the JavaDOC generator